index.js

```diff
@@ -14,13 +14,20 @@ module.exports = {
         "scss/at-import-no-partial-leading-underscore": true,
         "scss/at-mixin-argumentless-call-parentheses": "always",
         "scss/selector-no-redundant-nesting-selector": true,
-        "scss/at-rule-no-unknown": true,
+        "scss/at-rule-no-unknown": [
+            true,
+            { "ignoreAtRules": ["apply"] },
```

Comment:
nice. anything else we need to ignore due to tailwind?

Reply:
nope, I'm pretty sure that's all we need.